lib/ui/views/login/components/social_media_buttons.dart

```diff
@@ -1,30 +1,45 @@
 import 'dart:io';
 
+import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:nowu/services/auth.dart';
+import 'package:nowu/ui/views/login/bloc/login_event.dart';
 
-import '../login_viewmodel.dart';
+import '../bloc/login_bloc.dart';
 
 class SocialMediaLoginButtons extends StatelessWidget {
-  SocialMediaLoginButtons(this._viewModel);
+  SocialMediaLoginButtons(this._loginBloc);
 
-  final LoginViewModel _viewModel;
+  final LoginBloc _loginBloc;
 
   @override
   Widget build(BuildContext context) => Row(
         mainAxisAlignment: MainAxisAlignment.spaceAround,
         children: [
-          Platform.isIOS
+          !kIsWeb && Platform.isIOS
```

Comment:
This can be now part of the `LoginBlocState` right? Something like `showAppleLoginButton`